x-pack/plugins/cloud_security_posture/public/pages/findings/layout/findings_distribution_bar.tsx

```diff
@@ -18,11 +18,14 @@ import {
 import { FormattedMessage } from '@kbn/i18n-react';
 import { i18n } from '@kbn/i18n';
 import numeral from '@elastic/numeral';
+import { RULE_FAILED, RULE_PASSED } from '../../../../common/constants';
+import { Evaluation } from '../../../../common/types';
```

Comment:
```suggestion
import type { Evaluation } from '../../../../common/types';
```

^ for all identical imports